arch/arm/configs/bcm2711_defconfig

```diff
@@ -850,6 +850,7 @@ CONFIG_VIDEO_EM28XX_V4L2=m
 CONFIG_VIDEO_EM28XX_ALSA=m
 CONFIG_VIDEO_EM28XX_DVB=m
 CONFIG_V4L_PLATFORM_DRIVERS=y
+CONFIG_VIDEO_BCM2835_UNICAM=m
```

Comment:
Adding to the defconfig should be in a different patch, and should be enabled in bcmrpi, bcm2708 and bcm2711 defconfigs. Possibly arch/arm64/configs/ variants too.

Reply:
Done.

I have left out arch/arm64.  I want to test this on 64-bit build before adding by default.